### DISCUSSION TITLE
【招聘】SCP基金会招聘公告

### DISCUSSION THREAD

**Discussion by wjw-0629** [share121/inter-knot#3375] [其他]
![image](https://github.com/user-attachments/assets/28eb0ca3-992e-410b-9486-752d0051856f)
人类到如今已经繁衍了250000年，只有最近的4000年是有意义的。

所以，我们在将近250000年中在干嘛？我们躲在山洞中，围坐在小小的篝火边，畏惧那些我们不懂得的事物——那些关于太阳如何升起的解释，那些人头鸟身的怪物，那些有生命的石头。所以我们称他们为“神”和“恶魔”，并向他们祈求宽恕和祈祷拯救。

之后，他们的数量在减少，我们的数量在增加。当我们恐惧的事物越来越少，我们开始更理智的看待这个世界。然而，不能解释的事物并没有消失，好像宇宙故意要表现出荒谬与不可思议一样。

人类不能再生活在恐惧中。没有东西能保护我们，我们必须保护我们自己。

当其他人在阳光下生活时，我们必须在阴影中和它们战斗，并防止它们暴露在大众眼中，这样其他人才能生活在一个理智的，普通的世界中。
![image](https://github.com/user-attachments/assets/6b0e809d-b1b6-468d-b154-2fa66412c913)我们控制，我们收容，我们保护。— The Administrator

基金会诚招D级人员帮助█████博士对scp-173进行进一步研究
应聘要求：乐观，有良好抗压能力，身体素质好，敢于尝试，勇敢，有行动力，冷静
工作描述：基金会吉祥物之一的小花生最近心情好像不太好，你与你的工作小组成员（共计三人）需要进入并清理祂的房间，和祂一起玩木头人的游戏
人员待遇：预计工期短；待遇高；基金会提供专门服务，在您工作顺利结束后可以回归正常生活（前提是顺利结束）；官方背书，人员身份不限（或许你在工作结束后还能继续工作，你还可以去陪伴scp-682,scp-096,scp-106等小可爱们，scp-049也会悉心治疗照顾你的）

**Comment by PETERL136** [comment]
這委託我接了!

**Reply by ALXY-12** [reply]
把脚本关了,天天接不到委托。<img src="https://bbs-static.miyoushe.com/static/2024/07/31/33f69e7df0fb9d06592379f11d7ba6ba_1797154502027561525.png" alt="云璃 威慑" width="75" >

**Reply by lllliufeng** [reply]
合着我的委托给人机抢了？<img src="https://bbs-static.miyoushe.com/static/2024/06/19/a751ee1e27d4253ea0c233c905f36085_3464665030406935104.png" alt="托帕 眺望" width="75" >

**Reply by PETERL136** [reply]
不是哥們我活的

**Comment by cagbaby** [comment]
接委托还开挂，生怕自己的钱不够违约金的是吧

**Reply by PETERL136** [reply]
哥們啥掛?我對岸的不太懂
